app/src/main/java/com/kms/cura/view/activity/PatientBasicSettingsActivity.java

```diff
@@ -0,0 +1,190 @@
+    }
+
+    private void initSpinner() {
+        initSexSpinner();
+
+        initCountriesSpinner();
+
+        initInsuranceSpinner();
+    }
+
+    private void initInsuranceSpinner() {
+        ArrayList<String> insurances = new ArrayList<String>(Arrays.asList(this.getResources().getStringArray(R.array.Insurance)));
+        String currentInsurance = entity.getInsurance();
+        int position = 0;
+        for (int i = 0; i < insurances.size(); i++) {
+            if (currentInsurance.equals(insurances.get(i))) {
+                position = i;
+            }
+        }
+        StringListAdapter insuranceAdapter = new StringListAdapter(this, R.layout.string_list_item2, insurances, 0);
+        spnInsurance_settings = (Spinner) findViewById(R.id.spnInsurance_Settings);
+        spnInsurance_settings.setAdapter(insuranceAdapter);
+        spnInsurance_settings.setSelection(position);
+    }
+
+    private void initCountriesSpinner() {
+        ArrayList<String> countries = Countries.getAllCountries();
+        StringListAdapter countryAdapter = new StringListAdapter(this, R.layout.string_list_item2, countries, 0);
+        spnCountry_settings = (Spinner) findViewById(R.id.spnCountry_Settings);
+        spnCountry_settings.setAdapter(countryAdapter);
+    }
+
+    private void initSexSpinner() {
+        ArrayList<String> sex = new ArrayList<>();
+        sex.add(getString(R.string.male));
+        sex.add(getString(R.string.female));
+        StringSexListAdapter adapter = new StringSexListAdapter(this, R.layout.string_list_item2, sex);
+        spnSex_settings = (Spinner) findViewById(R.id.spnSex_Settings);
+        spnSex_settings.setAdapter(adapter);
+        if (entity.getGender() != null && entity.getGender().equals("F")) {
+            spnSex_settings.setSelection(1);
+        } else {
+            spnSex_settings.setSelection(0);
+        }
+    }
+
+    private void initEditText() {
+        name = (EditText) findViewById(R.id.editText_settings_name);
+        name.setText(entity.getName());
+
+        city = (EditText) findViewById(R.id.editText_settings_city);
+        String formatedCity = getFormatedCityString();
+        city.setText(formatedCity);
+    }
+
+    private String getFormatedCityString() {
+        String formatedCity = null;
+        try {
+            formatedCity = new String(entity.getLocation().getBytes("ISO-8859-1"), "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            city.setText("");
+        }
+        return formatedCity;
+    }
+
+    @Override
+    public void onClick(View v) {
+        if (v.getId() == R.id.button_settings_calendar) {
+            DatePickerDialog dateDialog = new DatePickerDialog(this, AlertDialog.THEME_HOLO_LIGHT, myDateListener, year, month - 1, day);
+            dateDialog.show();
+        } else if (v.getId() == R.id.button_settings_save) {
+            Toast.makeText(PatientBasicSettingsActivity.this, "Save", Toast.LENGTH_SHORT).show();
```

Comment:
This is not necessary since we'll remove this toast message when we implement the function anyway.